docs/quick_start.md

```diff
@@ -0,0 +1,84 @@
+# Quick Start
+
+!!! Note
+    Currently, using the OSTIS Metasystem natively on Windows isn't supported.
+
+1. Clone repository of the OSTIS Metasystem:
+   
+    ```sh
+    git clone https://github.com/NikitaZotov/ostis-metasystem
+    cd ostis-metasystem
+    git checkout feat/update_build_system
+    ```
+
+2. Install C++ problem solver:
+
+    ```sh
+    ./scripts/install_cxx_problem_solver.sh
+    ```
+
+3. Build knowledge base:
+
+    ```sh
+    ./install/sc-machine/bin/sc-builder -i repo.path -o kb.bin --clear
+    ```
+
+4. Run C++ problem solver:
+
+    ```sh
+    ./install/sc-machine/bin/sc-machine -s kb.bin -c ostis-metasystem.ini \
+        -e "install/sc-machine/lib/extensions;install/sc-component-manager/lib/extensions;install/scp-machine/lib/extensions;install/problem-solver/lib/extensions"
+    ```
+
+5. Download component specifications from repositories. Execute the following command in a running terminal:
+
+    ```sh
+    components init
+    ```
+
+6. Install sc-web component. Execute the following command in the running terminal:
+
+    ```sh
+    components install sc_web
```

Comment:
this one already tries to install sc-web, so step 7 shouldn't be present
![image](https://github.com/user-attachments/assets/fd65d836-2f20-42cf-a526-d8d3006f8f52)

Reply:
Step 6 doesn't build sc-web